gnat2goto/driver/driver.adb

```diff
@@ -60,15 +60,15 @@ package body Driver is
    end GNAT_To_Goto;
 
    procedure Initialize_CProver_Internal_Variables (Start_Body : Irep) is
-      Int_32_T : constant Irep := Make_Signedbv_Type (Ireps.Empty, 8);
+      Int_32_T : constant Irep := Make_Signedbv_Type (Ireps.Empty, 32);
```

Comment:
This commit probably needs squashing into the first commit in the series? If not, at least the parts of this commit that are undoing changes made in the first commit needs squashing.

Reply:
The entire thing will be squashed once reviews are through, there's no need for this to come in multiple commits